### PR TITLE
Add documentation for NuxtJS

### PR DESCRIPTION
**Pull request by Strift**
Hello,

I added the docs for generating the website with Nuxt as well as the link to the docs regarding trailing slashes.

Cheers,

### CODE REVIEW

File: docs/Static-Site-Generators.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ Let's figure out how different static site generators output static files for ro
 - Default output: `/myPath/index.html`
 - Configuration: [trailingSlash option](https://docusaurus.io/docs/docusaurus.config.js#trailing-slash)
 
+## NuxtJS
+
+- Command: `nuxt generate`
+- Default output: `/myPath`
```
Comment (slorber) [inline docs/Static-Site-Generators.md:26]:
```suggestion
- Default output: `/myPath.html`
```

I guess by default it outputs files with an .html extension 🤪 
Can you double-check that it's the correct file output when using `trailingSlash: false`? The doc is not explaining that.

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks for being the first contributor :)

File: docs/Static-Site-Generators.md (hunk 1 above)
Comment (Strift) [reply docs/Static-Site-Generators.md]: Oh, you're right.

Actually, it outputs a `/myPath/index.html` file.

It does the same with the default configuration (`trailingSlash: undefined`) or with `trailingSlash: false`.

Comment (slorber) [reply docs/Static-Site-Generators.md]: Thanks :)

**Pull request merged**